core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -25,6 +25,11 @@
     @ConfigItem(defaultValue = "false")
     public boolean uberJar;
 
+    /**
+     * Manifest configuration of the runner jar.
+     */
+    public ManifestConfig manifest;
```

Comment:
Ah sorry I missed that one but we need a `@ConfigItem` annotation even for a config group. It works OK for the config but we need that for the doc generation.